fhi_users/auth/rest_auth_views.py

```
@@ -354,8 +354,12 @@ def create(self, request: Request) -> Response:
         return super().create(request)
 
     def create_stripe_checkout_session(
-        self, email, professional_user_id, user_domain, continue_url, cancel_url
+        self, email, professional_user_id, user_domain, continue_url, cancel_url,
+        card_required=False,
     ):
+        payment_method_collection = "always"
+        if not card_required:
+            payment_method_collection = "if_required"
```

Ruff (0.8.2)

Local variable `payment_method_collection` is assigned to but never used

Remove assignment to unused variable `payment_method_collection`

(F841)